Lib/test/test_math.py

```diff
@@ -795,7 +795,8 @@ def testHypot(self):
         # Verify scaling for extremely large values
         fourthmax = FLOAT_MAX / 4.0
         for n in range(32):
-            self.assertEqual(hypot(*([fourthmax]*n)), fourthmax * math.sqrt(n))
+            self.assertTrue(math.isclose(hypot(*([fourthmax]*n)),
```

Comment:
Any reason to prefer this over [`assertAlmostEqual`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertAlmostEqual)?

Reply:
Yes.  *assertAlmostEqual()* only compares values rounded to seven places.  This is useless for testing very large or very small numbers.  

In contrast, [*math.isclose()*](https://docs.python.org/3/library/math.html#math.isclose) uses both a relative tolerance and absolute tolerance.  The relative tolerance is what we're after in this test.